### PR TITLE
updates checkin message when user clicks button

### PR DESCRIPTION
**Pull request by justineluo**
slack_iteraction_message_response

### CODE REVIEW

File: lambdaFiles/mainLambda.js [hunk 1]
```diff
@@ -79,11 +78,47 @@ function resendText(event, callback) {
...
+    });
+  });
+
+  req.on("error", (e) => {
+    console.error(e);
+  });
+
+  req.write(postData);
+  req.end();
+}
```
Comment (justineluo) [inline lambdaFiles/mainLambda.js:114]: this post request is quite ugly bc i can't use the `requests` or `axios` library ;-; 
perhaps we should invest time into trying to import npm modules for lambdas? (or maybe not bc we're almost done xd)
https://stackoverflow.com/questions/34437900/how-to-load-npm-modules-in-aws-lambda

Comment (zefwang) [reply lambdaFiles/mainLambda.js:117]: I think it's fine, the code is still understandable.

File: lambdaFiles/mainLambda.js [hunk 2]
```diff
@@ -79,11 +78,51 @@ function resendText(event, callback) {
...
 
+// updates the checkin message (kekw yes or no buttons) with a message that confirms their response
+// prevents spamming of kekw buttons bc buttons no longer accessible
+function updateCheckinMessage(responseUrl, user, buttonValue) {
+  var postData = JSON.stringify({
+    replace_original: "true", // edits original message
+    text:
+      buttonValue === "yes"
+        ? `${user.name} said you met! :happy-panda:`
+        : `${user.name} said you have not met! SADGE ;-; `, // TODO: show display name instead of user.name
```
[the remark this answers is not in the document]
Comment (justineluo) [reply lambdaFiles/mainLambda.js]: slack_iteraction_message_response
i did ittt

[the remark this answers is not in the document]
Comment (jia614) [reply lambdaFiles/mainLambda.js]: chungus

**Review by zefwang — APPROVED** [review]
I can't read so LGTM

**Pull request merged**